### PR TITLE
Add changelog for next release

### PR DESCRIPTION
**Pull request by t-b**
- [x] Wait for #370

### CODE REVIEW

**Comment by t-b** [comment]
Documentation is deployed at https://docs-staging.byte-physics.de/igor-unit-testing-framework.

**Review by Garados007 — CHANGES_REQUESTED** [review]
1. The following files have no version number:
    - `procedures\unit-testing-tracing-analytics.ipf`
    - `procedures\unit-testing-tracing-auto.ipf`
    - `procedures\unit-testing-tracing-tracer.ipf`
    - `procedures\unit-testing-tracing.ipf`
2. The following files have the old copyright year:
    - `License.txt`

**Comment by Garados007** [comment]
Do we want that the test files in `tests`, `internal_dev` and `docu\examples` have a version number as well?

**Comment by Garados007** [comment]
> I don't think so. What do you think?

These test files are bound to a specific version of UTF. We can highlight this if we just add the header `#pragma version=1.09` in the `.ipf` files. Updating this in the future is pretty easy as we only need to replace one number with the other:

```bash
find . -type f -name '*.ipf' -not -path '*/\.*' -exec sed -i 's/#pragma version=1.09/#pragma version=1.10/' {} +
```

**Comment by t-b** [comment]
> We can highlight this if we just add the header `#pragma version=1.09` in the `.ipf` files.

Okay. Done.

File: internal_dev/TestCaseAssertionCounterCheck.ipf [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #pragma TextEncoding = "UTF-8"
 #pragma rtGlobals=3		// Use modern global access method and strict wave access.
+#pragma version=1.09
 #pragma rtFunctionErrors=1
+#pragma version=1.09
```
[the remark this answers is not in the document]
Comment (t-b) [reply internal_dev/TestCaseAssertionCounterCheck.ipf]: Fixed.

**Review by Garados007 — APPROVED** [review]
LGFM

**Pull request merged**